R/etwfe.R

```diff
@@ -170,7 +177,7 @@ etwfe = function(
   }
   rhs = paste0(".Dtreat : ", rhs)
   
-  rhs = paste0(rhs, "i(", gvar, ", i.", tvar, ref_string, ")")
+  rhs = paste0(rhs, "i(", gvar, ", as_factor(", tvar, "), ", ref_string, ")")
```

Comment:
Please change this back to the original formulation. I don't see what switching out the idiomatic `i.` fixest syntax for `as_factor` is giving us besides another undeclared tidyverse dependency.